master/internal/api_experiment_intg_test.go

```diff
@@ -311,9 +311,15 @@ searcher:
 // nolint: exhaustivestruct
 func TestCreateExperimentCheckpointStorage(t *testing.T) {
 	api, _, ctx := setupAPITest(t, nil)
-	api.m.config.CheckpointStorage = expconf.CheckpointStorageConfig{}
+	oldConfig := api.m.config.Load()
+	newConfig := *oldConfig
+	newConfig.CheckpointStorage = expconf.CheckpointStorageConfig{}
+	api.m.config.CompareAndSwap(oldConfig, &newConfig)
```

Comment:
just use `config.Store` if you want to unconditionally store the new config (so use it everywhere in tests)

Reply:
Done